tests/create-rapid/enc/optee/user_ta_header_defines.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
```

Comment:
Auto-generate this file from the **uuid.txt** file in the same source directory and place it in **CMAKE_CURRENT_BINARY_DIR**. The auto-generation tool can be placed in **toosl/oeoptee** perhaps.

Reply:
Idem re. `OE_SET_ENCLAVE_OPTEE`.